_posts/2022-06-21-share-bold-ideas

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "Sharing Crazy Bold Ideas"
+description: "The impact of sharing crazy bold ideas"
+comments: true
+keywords: "sharing, crazy, bold, ideas"
+published: false
+---
+
+In 2007 I got into (Mfantsipim High School)[https://en.wikipedia.org/wiki/Mfantsipim_School] and my tech journey began. My school was nicked named Botwe, after the hill on which it is situated. We further shortened it to B-Town.  I started to pick up programming, I started with ruby and html. It was a boarding school and computers were not allowed. So we wrote out programs on scrap notes and typed them out on the computers when we got the chance. That was once a week. 
+
+Fast forward to 2 years later and I came up with this bold crazy idea to develop a video game for my high school. I called it B-Town Vice City. The concept was simple create a game very much like the much famed (Vice City)[https://en.wikipedia.org/wiki/Grand_Theft_Auto:_Vice_City] at the time with the setting as my  high school. I had less than a 500 hours programming experience. I knew nothing about video games or 3d modeling. Or anything for that matter. I told a bunch of guys in the computer club who also thought it was a bold idea but they liked it so we got to work trying to learn everything there was about making video games. 
+
+Word slowly went out that Bubu and the Computer Club were making a video game; B-Town Vice City. I became a laughing stock. Everywhere I went, people pointed fingers at me and laughed at me; That is the guy who wants to build Vice City. It became so bad that some guys in the club started to distance themselves from the effort because we had become the latest laughing stock. People would see me and ask me how my game was going basically to elicit laughter from their group of friends basically at my expense. It bothered me at first but nonetheless I persisted. I slowly began to grow a thick skin. I dont know where the conviction came from but I was set on developing a video game and nothing anyone would say would deter me. And for a long time nothing did. Now that I look back I think I would probably have laughed at me back then. 
```

Comment:
```suggestion
Word slowly went out that Bubu and the Computer Club were making a video game; B-Town Vice City. I became a laughing stock. Everywhere I went, people pointed fingers at me and laughed at me; That is the guy who wants to build Vice City. It became so bad that some guys in the club started to distance themselves from the effort because we had become the latest laughing stock. People would see me and ask me how my game was going basically to elicit laughter from their group of friends basically at my expense. It bothered me at first but nonetheless, I persisted. I slowly began to grow a thick skin. I don't know where the conviction came from but I was set on developing a video game and nothing anyone would say would deter me. And for a long time, nothing did. Now that I look back I think I would probably have laughed at myself back then. 
```